hydra-cluster/test/Test/EndToEndSpec.hs

```diff
@@ -122,7 +123,8 @@ spec = around showLogsOnFailure $ do
               let firstNodeId = clusterIx * 3
 
               hydraScriptsTxId <- publishHydraScriptsAs node Faucet
-              withHydraCluster tracer tmpDir nodeSocket firstNodeId cardanoKeys hydraKeys hydraScriptsTxId $ \nodes -> do
+              let contestationPeriod = UnsafeContestationPeriod 2
```

Comment:
is there a reason why this is 2 and not 1 as the rest?

maybe like in TUI we can create a commented defaultContestationPeriod to re-use.

Reply:
We are using 2 because this test was previously using the same value https://github.com/input-output-hk/hydra/pull/615/files/d91550eeb74deee5c4ee337cd3942872ffa3d39d#diff-f3f9817ebc889ef314514ecb08ab7ffec2d5454063e34d381c2b1c8b0daca830L134